### PR TITLE
70 routing based on IDs only

### PR DESCRIPTION
**Pull request by AlZorba**
**Issue** :  `https://github.com/archaiodata/thacer/issues/70`

**Description** :    
This PR introduce the use of unique IDs, hence removes the routes with ANA and INV. 
It also clean the display of Ceram page by hiding empty section.

### CODE REVIEW

File: vue-thacer/src/assets/js/thacer-map-setup-search.js [hunk 1]
```diff
@@ -32,11 +32,10 @@ export function setupSearchCeramByText(markerClusterGroupCeram, map) {
             doesCeramObjectPassesInputSearchString(obj.properties, inputSearchString)
           ) {
             let label
-            if (obj.properties.Pi != null) {
+            if (obj.properties.Pi) {
               label = 'Π' + obj.properties.Pi
             } else {
-              label = obj.properties.Inv_Fouille
-
+              label = obj.properties.ID
```
Comment (francoiscrouzet) [inline vue-thacer/src/assets/js/thacer-map-setup-search.js:38]: You may want to do some one liner in such case
let label = obj.properties.Pi ? 'Π' + obj.properties.Pi : obj.properties.ID
or even with ??
or just like it is now :)

File: vue-thacer/src/components/TheCeramicImages.vue [hunk 2]
```diff
@@ -1,17 +1,19 @@
 <template>
-  <div v-if="imageUrlArrayList !== null">
+  <div v-if="imageUrlArrayList.PHOTO.length != 0">
```
Comment (francoiscrouzet) [inline vue-thacer/src/components/TheCeramicImages.vue:2]: Ah, try to use `==!`
or just `>`

Another thing, are you sure that PHOTO always exist and is an array ? If not there could be type error. maybe `imageUrlArrayList.PHOTO?.length`

File: vue-thacer/src/components/TheCeramicImages.vue [hunk 3]
```diff
@@ -1,17 +1,19 @@
...
     <TheCeramicImagesSubGroup
       title="Photographie"
       :imageUrlArray="imageUrlArrayList.PHOTO"
       noImageMessage="il n'y a pas de photographie disponible pour cet exemplaire."
     ></TheCeramicImagesSubGroup>
-
+  </div>
+  <div v-if="imageUrlArrayList.THA_MACRO_PHOTO.length != 0">
```
Comment (francoiscrouzet) [inline vue-thacer/src/components/TheCeramicImages.vue:9]: same

File: vue-thacer/src/components/TheCeramicImages.vue [hunk 4]
```diff
@@ -1,17 +1,19 @@
...
     <TheCeramicImagesSubGroup
       title="Macrophoto"
       :imageUrlArray="imageUrlArrayList.THA_MACRO_PHOTO"
       noImageMessage="il n'y a pas de macrophoto pour cet exemplaire."
     ></TheCeramicImagesSubGroup>
-
+  </div>
+  <div v-if="imageUrlArrayList.Profils.length != 0">
```
Comment (francoiscrouzet) [inline vue-thacer/src/components/TheCeramicImages.vue:16]: Same

File: vue-thacer/src/components/TheCeramicImages.vue (hunk 2 above)
Comment (francoiscrouzet) [reply vue-thacer/src/components/TheCeramicImages.vue:2]: Ah it's here, trying to make sense of the `==` column  : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#comparing_equality_methods   🤯😱

File: vue-thacer/src/assets/js/thacer-map-create-layer.js [hunk 5]
```diff
@@ -1,67 +1,59 @@
...
+  if (ceramLayer.feature.properties.Description) {
+    popup = popup + ceramLayer.feature.properties.Description + '<br>'
+  }
+  if (ceramLayer.feature.properties.Biblio) {
+    popup = popup + ceramLayer.feature.properties.Biblio + '<br>'
+  }
+  if (ceramLayer.feature.properties.Publication) {
+    popup = popup + ceramLayer.feature.properties.Publication + '<br>'
+  }
+  ceramLayer.bindPopup(popup + '<a href=' + createCeramUrl(ceramLayer) + '>Fiche</a>', {
```
Comment (francoiscrouzet) [inline vue-thacer/src/assets/js/thacer-map-create-layer.js:49]: Super cool

**Review by francoiscrouzet — APPROVED** [review]
Yo ! Super cool refactor !
There is just the `console.log` you should remove, or if it's necessary, comment why it is here.
And eventually the `==!` instead of `=!`

I don't have the big picture of it, and I think you'll ever be the only one to understand these 3 ids system, but if you want you can tell me during a call.
I also understand that this PR doesn't imply any change in the backend or data structure?

File: vue-thacer/src/assets/js/thacer-map-setup-search.js (hunk 1 above)
Comment (AlZorba) [reply vue-thacer/src/assets/js/thacer-map-setup-search.js:38]: Very wise

File: vue-thacer/src/components/TheCeramicImages.vue (hunk 2 above)
Comment (AlZorba) [reply vue-thacer/src/components/TheCeramicImages.vue:2]: first one seems to work.
second, yes PHOTO always exist
Thanks

File: vue-thacer/src/assets/js/thacer-map-create-layer.js (hunk 5 above)
Comment (AlZorba) [reply vue-thacer/src/assets/js/thacer-map-create-layer.js:49]: :)

**Pull request merged**